_components/in-page-navigation/guidance/implementation.md

```diff
@@ -17,6 +17,8 @@ The following is an example of the minimum markup your page would need to inplem
 ```
 
 {:.usa-content-list }
+- **Root margin.** The root margin (`data-root-margin`) is a string that allows you to define the observable area. This set of values grows or shrinks the observable area from each side of the root element's bounding box before computing intersections. It can accept values similar to the CSS margin property, and can be percentages. By default, the in-page navigation component is set to `0px 0px 0px 0px`. 
```

Comment:
```suggestion
- **Root margin.** The root margin property (`data-root-margin`) is a string that allows you to define the observable area. This set of values grows or shrinks the observable area from each side of the root element's bounding box before computing intersections. It can accept values similar to the CSS margin property, and can be percentages. By default, the in-page navigation component is set to `0px 0px 0px 0px`. 
```